pkg/virt-handler/node-labeller/cpu_plugin.go

```diff
@@ -91,6 +94,11 @@ func (n *NodeLabeller) loadDomCapabilities() error {
 	usableModels := make([]string, 0)
 	for _, mode := range hostDomCapabilities.CPU.Mode {
 		if mode.Name == v1.CPUModeHostModel {
+			if virtconfig.IsARM64(runtime.GOARCH) {
```

Comment:
I am considering if we can put the architecture information into the `NodeLabeller struct`. Thus, if we have future architecture specific modification, we can easily get the architecture information, rather than use "runtime.GOARCH" everywhere, but we can work on it as a follow-up.

Reply:
ACK